src/main/java/org/folio/utils/InvoiceLineUtils.java

```diff
@@ -0,0 +1,81 @@
+package org.folio.utils;
+
+import lombok.experimental.UtilityClass;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang.time.DateUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.jaxrs.model.InvoiceLine;
+
+import java.util.Date;
+import java.util.Objects;
+
+@Log4j2
+@UtilityClass
+public class InvoiceLineUtils {
+
+  /**
+   * Checks if any invoice line fields (tags, subscription info/dates, comment) have changed.
+   * Returns true if changes detected, false if all fields are identical.
+   */
+  public boolean isIgnoreMissingBudgets(InvoiceLine storage, InvoiceLine request) {
+    // Example: storage=null, request=[x,y,z]
+    var hasTagsAdded = Objects.isNull(storage.getTags())
+      && Objects.nonNull(request.getTags())
+      && !CollectionUtils.sizeIsEmpty(request.getTags().getTagList());
+
+    // Example: storage=[x,y,z], request=null
+    var hasTagsRemoved = Objects.isNull(request.getTags())
+      && Objects.nonNull(storage.getTags())
+      && !CollectionUtils.sizeIsEmpty(storage.getTags().getTagList());
+
+    // Example: storage=[x,y,z], request=[x,y,w]
+    // Example: storage=[x,y,z], request=[]
+    // Example: storage=[], request=[x,y,z]
+    var hasTagsChanged = Objects.nonNull(storage.getTags())
+      && Objects.nonNull(request.getTags())
+      && !CollectionUtils.isEqualCollection(storage.getTags().getTagList(), request.getTags().getTagList());
+
+    var hasSubInfoChanged = !StringUtils.equals(storage.getSubscriptionInfo(), request.getSubscriptionInfo());
+
+    // Example: storage=null, request=2025-01-01
+    // Example: storage=2025-01-01, request=null (removed)
+    // Example: storage=2025-01-01, request=2025-01-02 (changed)
```

Comment:
I think comments are not needed in this utility class since logic is pretty simple except java doc